### ISSUE TITLE
[Feature]: input token logits

### ISSUE THREAD

**Issue by xs1997zju** [vllm-project/vllm#11106]
### 🚀 The feature, motivation and pitch

Does it support only inferring the input tokens without generating an answer, but returning the logits of each token of the input?

### Before submitting a new issue...

- [X] Make sure you already searched for relevant issues, and asked the chatbot living at the bottom right corner of the [documentation page](https://docs.vllm.ai/en/latest/), which can answer lots of frequently asked questions.
